api/src/main/java/demo/service/filter/StringCriteria.java

```diff
@@ -0,0 +1,28 @@
+package demo.service.filter;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum StringCriteria implements Filters {
+    CONTAINS,
+    DOES_NOT_CONTAIN,
+    EQUALS;
+
+    @Override
+    public String getCode() {
+        return this.name();
+    }
+
+    public static boolean isStringInEnumList(String string) {
```

Comment:
export as an util function and call it in every instance, too much duplicated code